grr/core/grr_response_core/lib/casing_test.py

```diff
@@ -0,0 +1,32 @@
+# Lint as: python3
+"""Tests for casing functions."""
+
+from grr_response_core.lib import casing
+
+from grr.test_lib import test_lib
+
+
+class CasingTests(test_lib.GRRBaseTest):
```

Comment:
`GRRBaseTest` is evil, `absltest.TestCase` is the way to go unless your test has no control over its dependencies and requires a tons of GRR-specific setup.

Reply:
Ow, thank you pointing this out! I made the change here and I'll keep it in mind in the future.